edx_exams/apps/core/signals/signals.py

```diff
@@ -80,3 +85,28 @@ def emit_exam_attempt_rejected_event(user, course_key, usage_key, exam_type):
             exam_type=exam_type
         )
     )
+
+
+def emit_exam_attempt_errored_event(user, course_key, usage_key, exam_type):
+    """
+    Emit the EXAM_ATTEMPT_ERRORED Open edX event.
+    """
+    user_data = UserData(
```

Comment:
at some point (either this PR or later) we should probably make a helper function to create this object since it's always the same

Reply:
Created a helper function to create the UserData object